packages/edge-login-ui-rn/src/components/themed/BackButton.js

```diff
@@ -12,36 +12,61 @@ import {
   withTheme
 } from '../services/ThemeContext'
 
+const isIos = Platform.OS === 'ios'
+
 type Props = {
   onPress(): void,
-  marginRem?: number[] | number
+  marginRem?: number[] | number,
+  enabled?: boolean
 }
 
 const BackButtonComponent = ({
   theme,
   marginRem,
-  onPress
+  onPress,
+  enabled = true
 }: Props & ThemeProps) => {
   const styles = getStyles(theme)
   const spacings = sidesToMargin(mapSides(fixSides(marginRem, 0.5), theme.rem))
 
   return (
     <View style={[spacings, styles.container]}>
-      <TouchableWithoutFeedback onPress={onPress}>
-        <SimpleLineIcons
-          name="arrow-left"
-          size={theme.rem(0.8)}
-          color={theme.icon}
-        />
-      </TouchableWithoutFeedback>
+      {enabled && (
+        <TouchableWithoutFeedback onPress={onPress}>
+          {isIos ? (
+            <IonIcon
+              size={theme.rem(1.5)}
+              name="chevron-back-outline"
+              style={styles.backIconStyle}
+            />
+          ) : (
+            <IonIcon
+              size={theme.rem(1.25)}
+              name="md-arrow-back"
+              style={styles.backIconAndroid}
+            />
+          )}
+        </TouchableWithoutFeedback>
+      )}
     </View>
   )
 }
 
 const getStyles = cacheStyles((theme: Theme) => ({
   container: {
-    width: theme.rem(0.8),
-    height: theme.rem(0.8)
+    flexDirection: 'row',
+    alignItems: 'center',
```

Comment:
Also if you need centered content on the vertical axis no need for `flexDirection: 'row' with alignItems: 'center'` will only be enough `justifyContent: 'center'`